components/expander.jsx

```diff
@@ -32,7 +32,7 @@ var Expander = React.createClass({
     return (
       <div className="expander-container">
         <div className={className} tabIndex="0" onBlur={this.collapse} onFocus={this.expand}>
-          <div onMouseDown={this.onClick} className="expander-header">
+          <div onClick={this.handleClick} className="expander-header">
```

Comment:
https://facebook.github.io/react/docs/events.html#mouse-events

I'm not sure why but handler of `onMouseDown` doesn't return me an `event` object in callback.  Changing the event to `onClick` seem to solve the problem.

Also, wondering if `e.preventDefault();` in the handler is really needed? or maybe we should this from `<div>` to `<a>` to indicate this is something togglable.